### DISCUSSION TITLE
Nixie Controller - External Override to disable Schedule

### DISCUSSION THREAD

**Discussion by brentk7** [tagyoureit/nodejs-poolController#994] [Nixie]
I am currently using a Nixie controller to control my filter pump and salt chlorinator. I also have njsPC connected to Home Assistant. I would like to prevent the scheduler from running (or stop the pump) based on a signal from Home Assistant when my solar production is below a certain value (only run the pump when solar production covers the power).
I would rather not use an automation from HA to stop and start the pump, so that if the communication between HA and njsPC is down, then the njsPC schedule will take priority and control the pump.

Is there any way to achieve this functionality?

**Comment by tagyoureit** [comment]
You can call the api's for the schedule to disable/enable them.  But if the communication is down you'd need something running locally on the njsPC machine to monitor that and re-enable the schedule.  

Otherwise, a simpler solution would be to just stop the pump if the solar production is low and it's currently on.  If you want to re-enable it, you'd need additional logic to check all the schedules to see if the pump should be on and the solar is producing enough juice.

**Reply by brentk7** [reply]
Thanks, you are right - it would probably be better to just stop and start the pump. Is there a way  from within either njsPC or REM to have the state of something like an MQTT variable enable or stop and start the pump? I see both of them have an interface to MQTT, but I haven't been able to find any information regarding what can be done with MQTT or how.
